README.md

```diff
@@ -2,10 +2,38 @@
 
 This extension allow merchants to have cart gifts (free items, etc).
 
+## Configuration
+
+Set the following values in your Shopgate Connect Admin:
+* `endpoint` - (text) endpoint URL to fetch configuration
+* `configTTL` - (text) configuration TTL im seconds
+
+The endpoint should provide a JSON file:
+
+```json
+[ 
+  { 
+    "expression": "totals[.type == 'grandTotal'].amount > 10 && totals[.type == 'grandTotal'].amount <= 20", 
```

Comment:
we should inform the merchant about the changed config, because they still think it has to look like 
`"> 10.99", "<= 15"`
even though i like your idea to make it really flexible, can we maybe offer a more simple and easy to use solution?
maybe a basic and advanced config?

Reply:
They failed again generating the JSON for old spec, so we can give them an update.
I like the spec changes and as CS is setting it up, I think we don't have to simplify. Also in case the merchant generates it on their end: If they are capable of generating that file, they should be capable of using those "advanced" expressions too.
So I won't add more code complexity here for now.